src/ledger/LedgerTxn.cpp

```diff
@@ -1980,13 +1980,16 @@ LedgerTxn::Impl::WorstBestOfferIteratorImpl::clone() const
 size_t const LedgerTxnRoot::Impl::MIN_BEST_OFFERS_BATCH_SIZE = 5;
 
 LedgerTxnRoot::LedgerTxnRoot(Database& db, size_t entryCacheSize,
-                             size_t prefetchBatchSize)
-    : mImpl(std::make_unique<Impl>(db, entryCacheSize, prefetchBatchSize))
+                             size_t prefetchBatchSize,
+                             bool bestOfferDebuggingEnabled)
```

Comment:
Nitpick: there should be no mention of best offer debugging if it isn't compiled in. That will make it obvious that the behavior is unchanged if the flag isn't set, and prevent warnings about unused variables.

Reply:
Done.